scripts/python/PyAutoscoper/connect.py

```diff
@@ -103,6 +106,30 @@ def _openConnection(self):
         s.connect((self.address, 30007))
         return s
 
+    def _checkVersion(self):
+        """
+        Internal function, should not be called by a user.
+
+        Checks that the server version is compatible with the client version.
+
+        Called automatically upon init.
+
+        :raises AutoscoperServerError: If the server version is not compatible with the client version
+        """
+        response = self._send_command(0x10)  # 16
+        server_version = float(response[1:].decode("utf-8").strip("\x00"))
```

Comment:
Considering using `packaging.version` to parse and then compare version

See https://packaging.pypa.io/en/stable/version.html